DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CentreConfiguration/AddRegistrationPromptSummaryViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CentreConfiguration

```

Comment:
Re folder structuring - should we add a /RegistrationPrompts folder under centre configuration like we do for views to keep things consistent?

Reply:
I was going to sort that in the next ticket (381) that I haven't put for review yet as it needs methods from this one. Should I also move them to /Centre/Reg.... rather than /CentreConfiguration/Reg... at the same time?

Reply:
Yes please! But as part of a new ticket sounds good! 👍